lib/response.js

```diff
@@ -152,6 +152,9 @@ res.send = function send(body) {
     case 'boolean':
     case 'number':
     case 'object':
+    case 'bigint':
```

Comment:
It seems strange to add a case here for the type and then still check the type again in the case body. Should this not simply be it's own case and body with a break?

Reply:
adding bigint seperatlely would have made more sense! got it!